department-of-veterans-affairs/abd-vro
# Refactor - rename files for NEW condition LUT

## What was the problem?
 - just using "dropdown" in the code can be very confusing
 - the lookup table referred to dropdown corresponds to the dropdown field from the 526 frontend where veterans enter their condition(s)

Associated tickets or Slack threads:
- n/a

## How does this fix it?[^1]

- include condition throughout the code to make things more obvious what this lookup table is for

## How to test this PR
- navigate to `domain-cc/cc-app/` and run `pytest`


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
